app/uk/gov/hmrc/platforminitiatives/services/PlatformInitiativesService.scala

```diff
@@ -99,6 +99,18 @@ class PlatformInitiativesService @Inject()(
         version               = Version(2,13,0,"2.13.0"),
         team                  = team,
         experimental          = true
+      ),
+      createMigrationInitiative(
+        initiativeName        = "Depreciate Simple-Reactivemongo: HMRC-Mongo Migration",
+        initiativeDescription = s"Monitoring [repos still using simple-reactivemongo](" + url"https://catalogue.tax.service.gov.uk/dependencyexplorer/results?group=uk.gov.hmrc&artefact=simple-reactivemongo&team=$teamName&flag=production&scope=compile&versionRange=[0.0.0,99.0.0)&asCsv=false".toString.replace(")", "\\)") + " ) | [Confluence](" + url"https://confluence.tools.tax.service.gov.uk/display/TEC/2021/03/04/HMRC+Mongo+is+now+available" + ").",
```

Comment:
worth adding a link to "repos now using hmrc-mongo" too?

Reply:
Agreed - Will do this.